spec/Gaufrette/StreamWrapperSpec.php

```diff
@@ -200,21 +200,21 @@ function it_does_not_get_stat_when_is_not_open()
      */
     function it_stats_file(Stream $stream)
     {
-        $stat = array(
-            'dev'   => 1,
-            'ino'   => 12,
-            'mode'  => 0777,
+        $stat = [
+            'dev' => 1,
+            'ino' => 12,
+            'mode' => 0777,
             'nlink' => 0,
-            'uid'   => 123,
-            'gid'   => 1,
-            'rdev'  => 0,
-            'size'  => 666,
+            'uid' => 123,
```

Comment:
IMHO would be better to keep things align here

Reply:
Actually I'm not fan of aligning things, as it will make changes on other lines as soon as a big line is introduced ; and thus, making a new commit for these lines, whereas they were not significantly changed.
To me, it's the same issue than the trailing comma at the end of array items. Before if was supported, adding a new item to a list was producing a git diff of two lines : the actual line and the above one to add a comma. Only the actual line had a sense for the git diff.

That's why I prefer not to keep things aligned. Only alphabetic sort should improve readability IMHO.